frameworks/Java/micronaut/common/src/main/java/benchmark/controller/AbstractBenchmarkController.java

```diff
@@ -0,0 +1,57 @@
+package benchmark.controller;
+
+import benchmark.model.Fortune;
+import benchmark.model.World;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class AbstractBenchmarkController {
+
+    protected final Integer[] boxed = IntStream.range(1, 10001).boxed().toArray(Integer[]::new);
+
+    protected List<Fortune> createFortunes() {
+        List<Integer> fortuneMessages = IntStream.range(0, 10).boxed().collect(Collectors.toList());
+        List<Fortune> fortunes = new ArrayList<>(fortuneMessages.size());
```

Comment:
Same note here. This should be dynamically sized.

Reply:
This creates the test data for the unit testing